be/src/exec/tablet_sink.cpp

```diff
@@ -585,6 +588,7 @@ Status OlapTableSink::init(const TDataSink& t_sink) {
 }
 
 Status OlapTableSink::prepare(RuntimeState* state) {
+    _span->AddEvent("prepare");
```

Comment:
Could it add query_id/load_id to trace a query or load?

Reply:
I already add `label`